svf/lib/WPA/Andersen.cpp

```diff
@@ -144,9 +125,30 @@ void AndersenBase::analyze()
         SVFUtil::stopAnalysisLimitTimer(limitTimerSet);
 
         DBOUT(DGENERAL, outs() << SVFUtil::pasMsg("Finish Solving Constraints\n"));
+}
+
+/*!
+ * Andersen analysis
+ */
+void AndersenBase::analyze()
+{
+    /// Initialization for the Solver
```

Comment:
if(!Options::ReadAnder().empty()){
   readPtsFromFile(Options::ReadAnder());
}
else{
    if(Options::WriteAnder().empty()) {
        initialize();
        solveConstraints();
        finalize();
    }
    else{
      solveAndwritePtsToFile(Options::WriteAnder());
    }
}


readPtsFromFile(filename){

initialize();
this->readFromFile(Options::ReadAnder());
PointerAnalysis::finalize();
}